pkg/common/args.go

```diff
@@ -28,6 +28,8 @@ import (
 	"sigs.k8s.io/hydrophone/pkg/log"
 )
 
+var containerImage string
```

Comment:
I'm torn on whether to move this into args.go or keep it in const.go so that all common variables are collocated, thoughts?